src/FeeCollector.sol

```diff
@@ -92,10 +97,14 @@ contract FeeCollector is Ownable, IFeeCollector {
     **
     ******************************************************************************/
 
+    function setFeeRate(uint256 _feeRate) public payable onlyOwner {
+        if (_feeRate > 1000) revert OutOfRange();
+        feeRate = _feeRate;
+    }
+
     /// @inheritdoc IFeeCollector
     function setClientRate(uint256 _clientRate) public payable onlyOwner {
-        if (_clientRate < 30 || _clientRate > 100) revert OutOfRange();
-
+        if (_clientRate > 100) revert OutOfRange();
```

Comment:
What is the rationale for not guaranteeing clients at least 30% of the protocol fee?

I understand that this gives us maximum optionality, but I think that it was quite reasonable to guarantee client operators a minimum percentage. I think the narrative of saying, "client operators are guaranteed X% of the protocol fee" is quite compelling to get people to build frontends.

It may make sense to lower it from 30%, but I do not think it should be 0%.

Reply:
On the one hand, I see it as unneeded complexity. I think market forces will be sufficient for whether or not frontends will want to integrate or not in the same way they are sufficient for all business we interact with on a daily basis. Hard-coding 30% (or any number) feels arbitrary. If the number is reduced to zero, frontends won't have an incentive to operate.

On the other hand, the hard-coded figure serves as a protection against future centralization, which I do like. I can add it back.

Reply:
Yeah, 30% feels arbitrary.

I still feel that the minimum should be non-zero. 20% makes more sense. That would be an 80-20 split.

...and we still have the freedom to change the fee rate, which could increase the revenue for both client operators and us.